Source/Web/app/HeatMap.js

```diff
@@ -270,11 +290,12 @@ function requestedIds() {
     }
   }
   
-  return result;
+  return result; /* Returns the IDs of interest */
 }
 
-var first = true;
-var heatmap;
+var first = true; /* Setting the first variable true for the if statement inside*/
```

Comment:
I should have made a comment for what this does, but anyway. "Set's to false once some co-ordinate information is found. It'll used the first point of the heatmap and set the map's camera to that position"